test/end-to-end/pages/signout/signout.js

```diff
@@ -1,14 +1,9 @@
 'use strict';
 
 const pageUnderTest = require('app/steps/ui/signout');
```

Comment:
Again, this object was missed in performance work, perhaps we should include this to avoid the re-introduction of the 'load the application object' pattern